music21/harmony.py

```diff
@@ -1321,6 +1326,47 @@ def removeChordSymbols(chordType):
     '''
     del CHORD_TYPES[chordType]
 
+def _getAlteration(alteration):
+    """
+    Parses the string-encoded alteration and returns a tuple containing the
+    alteration type (add, subtract/omit, alter), the degree of the chord,
+    and the number representing the chromatic alteration of this degree. add
+    is considered by default if not explicitly mentioned. If list is emtpy,
+    returns None.
+
+    >>> harmony._getAlteration('add4')
+    ('add', 4, 0)
+
+    >>> harmony._getAlteration('subtract3')
+    ('subtract', 3, -1)
+
+    >>> harmony._getAlteration('addb9')
+    ('add', 9, -1)
+
+    >>> harmony._getAlteration('add#9')
+    ('add', 9, 1)
+
+    >>> harmony._getAlteration('b9')
+    ('add', 9, -1)
+
+    >>> harmony._getAlteration('alter#5')
+    ('alter', 5, 1)
+
+    >>> harmony._getAlteration('alter##5')
+    ('alter', 5, 2)
+
+    """
+    if alteration != '':
```

Comment:
short circuit the loop here.  `if alteration == '': return None`

Reply:
Or better: `if not alteration: return None`

Reply:
Done!